charts/kube-prometheus-stack/templates/prometheus/rules-1.14/alertmanager.rules.yaml

```diff
@@ -29,7 +29,7 @@ spec:
     - alert: AlertmanagerFailedReload
       annotations:
         description: Configuration has failed to load for {{`{{`}} $labels.namespace {{`}}`}}/{{`{{`}} $labels.pod{{`}}`}}.
-        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-alertmanagerfailedreload
+        runbook_url: https://runbooks.prometheus-operator.dev/runbooks/alertmanager/alertmanagerfailedreload
```

Comment:
not sure if this should be hardcoded?

Reply:
Me neither, but this seems to be the choice upstream if I understand correctly (have seen this in other places as well). I could skip those changes, but this will be something to be tackled in general when the script is being invoked

Reply:
There is always the possibility of me messing something up :smile:, but when I run the script locally this is what it was produced. If we don't want to tackle this now I can just keep the changes in the script, that add the flags for burnrate and histogram rules and the respective changes in the templated files

Reply:
Could add something like this:
```
    'https://runbooks.prometheus-operator.dev/runbooks': {
        'replacement': '{{ .Values.defaultRules.runbookUrl }}',
        'init': ''},
```
Which produces a diff like the following:
```
-        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-alertmanagerconfiginconsistent
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}/alertmanager/alertmanagerconfiginconsistent
```